src/parent.ts

```diff
@@ -73,15 +73,16 @@ export class ClasschartsParentClient extends ClasschartsClient {
 
     this.sessionId = sessionID.session_id;
 
-    let pupil = await this.getPupil();
+    let pupil = await this.getPupils();
 
     this.studentId = pupil[0].id;
     this.studentName = pupil[0].pupil_name;
   }
   /**
    * Get Pupil details
+   * @returns an array fo Pupils connected to this parent's account
    */
-    async getPupil(options?: GetActivityOptions): Promise<ActivityResponse> {
+    async getPupils(): Promise<ActivityResponse> {
```

Comment:
Sorry to nag, but it's just this line again. The typings are saying that this function returns [ActivityResponse](https://github.com/Classcharts-API/classcharts-api/blob/0593f41f8b7d4850a847cf0cf3f5512e161c4e7c/src/types.ts#L105) which doesn't appear to be the correct typings for this function :D Correct me if I'm wrong